crates/host/src/metrics.rs

```diff
@@ -16,16 +16,12 @@ pub struct HostMetrics {
     // but we don't really hve a way of getting at those. We should figure out a way to get at that
     // information so we don't have to duplicate it here.
     pub host_id: String,
-    /// The host's lattice ID.
-    // Eventually a host will be able to support multiple lattices, so this will need to either be
-    // removed or metrics will need to be scoped per-lattice.
-    pub lattice_id: String,
```

Comment:
Leaving comment but will remove if I find it, should we still annotate metrics with the lattice?

Reply:
Yes we should, and you do! Found it 😄